packages/core/admin/admin/src/pages/SettingsPage/pages/ApiTokens/EditView/components/BoundRoute/index.js

```diff
@@ -33,7 +33,7 @@ function BoundRoute({ route }) {
           .{action}
         </Typography>
       </Typography>
-      <Stack horizontal hasRadius background="neutral0" borderColor="neutral200" spacing={0}>
+      <Flex horizontal hasRadius background="neutral0" borderColor="neutral200" gap={0}>
```

Comment:
Am I right in saying `horizontal` shouldn't be passed as a prop to <Flex> ?

Reply:
Nice catch. Won't do anything but I will remove!

Reply:
👍🏻 I think there are a few more cases in the code:

- 	packages/core/admin/admin/src/content-manager/components/RepeatableComponent/components/Component.js
- 	packages/core/admin/admin/src/content-manager/pages/EditSettingsView/components/DynamicZoneList.js
- 	packages/core/admin/admin/src/content-manager/pages/EditView/Header/index.js
- 	packages/core/admin/admin/src/pages/SettingsPage/pages/Roles/EditPage/index.js

Reply:
I should have written a codemod.